cmd/main.go

```diff
@@ -70,6 +71,9 @@ func main() {
 		log.Fatal("No targets configured")
 	}
 
+	// enable InsecureSkipVerify
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
Could we make this configurable? I think the default should be to verify but if you want to disable I don't mind.